cmd/mattermost/commands/config.go

```diff
@@ -140,6 +149,29 @@ func configGetCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func configShowCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	// check that no arguments are given
+	err = cobra.NoArgs(command, args)
+	if err != nil {
+		return err
+	}
+
+	// set up the config object
+	// Note: app.Config() returns a pointer value so make appropriate changes
```

Comment:
Can you elaborate on this comment? Not sure what it's trying to say.

Reply:
Sorry about this comment, this was a note to myself while I was writing this out. I will remove it and make the other changes as you requested.